spec/unit/local_system_spec.rb

```diff
@@ -42,6 +42,27 @@
     end
   end
 
+  describe ".architecture" do
+    before(:each) do
+      expect_any_instance_of(OsInspector).to receive(:inspect) do |instance, system, description|
```

Comment:
Unused block argument - `instance`. If it's necessary, use `_` or `_instance` as an argument name to indicate that it won't be used.<br>Unused block argument - `system`. If it's necessary, use `_` or `_system` as an argument name to indicate that it won't be used.